### PR TITLE
Merge 'sarabensahbbat_sol' branch to 'dev' branch.

### PR DESCRIPTION
**Pull request by SaraBenShabbat**
Wait for a trainer approval.

### CODE REVIEW

File: strings.py [hunk 1]
```diff
@@ -10,18 +10,34 @@
...
+    if ' ' in a_string:
+        str_with_no_duplicates += ' '
+
+    for char in list(string.ascii_lowercase):
+        if char in a_string:
+            str_with_no_duplicates += char
+
+    return str_with_no_duplicates
+
+print('---The string with no duplicates:\n' + no_duplicates('monty pythons flying circus'))
```
Comment (lidorg-dev) [inline strings.py:18]: return ''.join(sorted(set(a_string)))

**Review by lidorg-dev — CHANGES_REQUESTED** [review]
please fix the code

File: StateCap.py [hunk 2]
```diff
@@ -71,29 +71,44 @@
...
-        str += (key + ' , ' )
+    for key,value in STATES_CAPITALS.items():
+        str += (key + ' -> ' +  value + ' , ')
     print(str)
 
 def get_state(capital):
-    str = ''
+    state_lst = []
     for key, value in STATES_CAPITALS.items():
         if value == capital:
```
Comment (lidorg-dev) [inline StateCap.py]: please use lambda will be better

**Comment by SaraBenShabbat** [comment]
Fixing lambda expression.

**Review by lidorg-dev — APPROVED** [review]

**Comment by SaraBenShabbat** [comment]
Bs"d all is works !

**Pull request merged**